rollup.config.js

```diff
@@ -21,12 +37,19 @@ const commonBanner = licensePlugin({
   },
 })
 
+const env = {
+  fpcdn: getEnv('FPCDN', 'fpcdn.io'),
+  ingressApi: getEnv('INGRESS_API', 'api.fpjs.io'),
```

Comment:
I know it is a little late to discuss this, but, I find `ingressApi` variable a little too board. We could pick a more specific value, such as `ingressApiHost`. The benefit is that, if we were to add more configurable fields related to ingress API, we would have easier time. The same goes for `fpcdn` as well, it would be better to rename it as `fpCDNHost`.

Given the fact that other proxy integrations use `fpcdn` and `ingressApi`, I'd say let's keep it that way but keep it in mind for future.